### PR TITLE
feat: adds avoid-dispatching-multiple-actions-sequentially rule

### PR DESCRIPTION
**Pull request by jsaguet**
Based on ngrx-tslint-rules

### CODE REVIEW

File: tests/rules/avoid-dispatching-multiple-actions-sequentially.test.ts [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+export class FixtureComponent {
+  name = 'Tim'
+  constructor(private store: Store<{}>){}
+
+  ping() {
+    this.store.dispatch({ type: 'PING' })
+  }
+}`,
+  ],
+  invalid: [
```
Comment (timdeschryver) [inline tests/rules/avoid-dispatching-multiple-actions-sequentially.test.ts:20]: Could you create each invalid block as a separate snippet?
That way, it's a bit easier to read imho.

Comment (jsaguet) [reply tests/rules/avoid-dispatching-multiple-actions-sequentially.test.ts:20]: Will do !

**Comment by jsaguet** [comment]
@timdeschryver
I split invalid test cases.

I'll try to add more rules. I'm sure the community needs this eslint-plugin to ease the migration from tslint to eslint :)

**Comment by timdeschryver** [comment]
Thanks @jsaguet !

I noticed, this repo isn't set up for automated tests / releases.
I'll add it probably somewhere next week, so these rules can be released.

Feel free to ping me if needed.

**Pull request merged**